windows/src/global/delphi/general/compile.pas

```diff
@@ -265,6 +266,44 @@ function CompileKeyboardFileToBuffer(kmnFile: PChar; buf: PFILE_KEYBOARD; Compil
   Result := ckf(PAnsiChar(AnsiString(kmnFile)), buf, CompilerWarningsAsErrors, WarnDeprecatedCode, Callback, Target);  // I3310   // I4865   // I4866
 end;
 
+type
+  TKeyman_Diagnostic = procedure(mode: Integer); stdcall;
+
+procedure Compiler_Diagnostic(mode: Integer);
```

Comment:
Does l. 304 end up using `mode`? I would've expected `keyman_diagnostic(mode);`.

Reply:
Good catch. Resolved soon -- in the next PR.